src/olympia/reviewers/serializers.py

```diff
@@ -192,6 +192,9 @@ def _fetch_entries():
         return self._entries
 
     def get_entries(self, obj):
+        file_only = self.context.get('file_only', False)
+        if file_only:
```

Comment:
If you don't use the variable anywhere else, you could directly use `self.context.get` in the `if` statement.

Reply:
Given that we are only excluding entries, I would rather name the variable `exclude_entries`. That seems a bit more informative because what does "[including a] file only" mean for a "file serializer"?